src/structures/TextInputComponent.js

```diff
@@ -153,8 +153,8 @@ class TextInputComponent extends BaseMessageComponent {
   }
 
   /**
-   * Sets the value of this input text component
-   * @param {string} value Value of this input text component
+   * Sets the value of this text input component
+   * @param {string} value Value of this text input component
    * @returns {TextInputComponent}
    */
   setValue(value) {
```

Comment:
I think it would be better to change `.setValue()` to `.setDefaultValue()` to avoid confusion about what the final value will be, I think it is a matter of interpretation. If you say default, you imply that it is default, but it can change.

```suggestion
  setDefaultValue(value) {
```